app/models/user.rb

```diff
@@ -82,15 +82,15 @@ def self.find_or_create_from_cis2_oidc(userinfo)
   end
 
   def is_medical_secretary?
-    return false unless Flipper.enabled?(:cis2)
+    return email.include?("admin") unless Flipper.enabled?(:cis2)
```

Comment:
This is one of those times that I find `unless` kinda hard to read. Wish there was a `Flipper.disabled?`